Host a Static Website with Hugo and NGINX.md

```diff
@@ -496,29 +495,23 @@ A better approach is to use the pkill command.
 
     pkill hugo
 
-Then show the list of hugo processes.
-
-    ps aux | grep hugo
-
-Which should return output:
+Now `ps aux | grep hugo` should show no hugo servers running:
 
     username 16628  0.0  0.1  14224  1020 pts/1    S+   15:29   0:00 grep --color=auto hugo
 
 ### Web Browser Returns Bad Gateway
 
-Bad Gateway can be returned by the web browser if the Hugo server is not running or is not running at port 1313. 
+"Bad Gateway" can be returned by the web browser if the Hugo server is not running or is not running at port 1313. 
 
 <p align="center">
   <img src="/images/BadGateway.jpg" alt="Bad Gateway web page" /> 
 </p>
 
-One way to find out more about status of the Hugo server is to issue command:
-
-    ps aux | grep hugo
+In this case you can use `ps aux | grep hugo` to verify the server is not running.
```

Comment:
No need to repeat the info already given just above.

Reply:
Agreed.